src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-customtag-update.js

```diff
@@ -124,11 +124,11 @@ export default class EzBtnCustomTagUpdate extends EzWidgetButton {
      * Renders the option.
      *
      * @method renderChoice
-     * @param {String} choice
+     * @param {Array} choice
      * @return {Object} The rendered option.
      */
-    renderChoice(choice) {
-        return <option value={choice}>{choice}</option>;
+    renderChoice(choice, label) {
+        return <option value={choice[0]}>{choice[1]}</option>;
```

Comment:
This seems like something is wrong here. You got an array with two elements where first is value and second is label?

Reply:
Yes, you are right. `label` param is redundant and is removed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1001/commits/a1988e287532c2931bfceac757b8eb954ffd1b51

Reply:
Now I know why you had the `label` param and probably this will be more readable:
```js
renderChoice([value, label]) {
    return <option value={value}>{label}</option>;
}
```

Reply:
Thanks, it is changed: https://github.com/ezsystems/ezplatform-admin-ui/pull/1001/commits/a733582383ac5012d63c8862e8ea9f7d23077180